Runtime/CourseController/Controllers/UIBaseCourseController.cs

```diff
@@ -25,13 +27,16 @@ public virtual GameObject GetCourseMenuPrefab()
         /// <inheritdoc />
         public override List<Type> GetRequiredSetupComponents()
         {
-            return new List<Type> {typeof(CourseMenuSpawner)};
+            return new List<Type> {typeof(CourseMenuSpawner), typeof(PlayerInput)};
```

Comment:
Does every UIBaseCourseController really need player input? I think it depends on the controller itself. Standalone for example does not need it